sonaranalyzer-dotnet/src/SonarAnalyzer.Common/Helpers/ProjectTypeHelper.cs

```diff
@@ -50,6 +50,7 @@ public static bool IsTest(this Compilation compilation)
         private const string TestAssemblyNamePattern = "TEST";
 
         private static readonly ISet<string> TestAssemblyNames = ImmutableHashSet.Create(

```

Comment:
I guess this shouldn't be part of this change but we could use `HashSet` instead of `ImmutableHashSet` as we saw performances are better.